fastlane/lib/before_all.rb

```diff
@@ -1,7 +1,8 @@
 before_all do
   case lane_context[:PLATFORM_NAME]
   when :ios
-    setup_travis
+    setup_travis if travis?
```

Comment:
Weird edge case here if both `travis?` and `circle?` return true, but I don't think we need to worry about that.

Reply:
I don't think we need to worry about that, unless Circle decides to try and pretend to be Travis.

Plus, from skimming the source ([`setup_travis`][t], [`setup_circle_ci`][c]), it looks like they might not even conflict with each other. Maybe the keychain setup would, IDK. 

tl;dr I don't think we need to _ensure_ that we only run one of them.